pylucene/app/adapter.py

```diff
@@ -92,13 +93,16 @@ def index_issues(self,
         )
         issues = repo.search(
             query,
-            attributes=['key', 'summary', 'description']
+            attributes=['key', 'summary', 'description',"comments"]
         )
         predictions = {}
         if model_id is not None:
-            model = repo.get_model_by_id(model_id)
-            predictions = model.get_version_by_id(version_id).predictions
-
+            predictions = requests.get(f"http://172.30.0.1:8000/models/{model_id}/versions/{version_id}/predictions",
+                json={
+                    'issue_ids': [i.identifier for i in issues]
+                })
+            predictions = predictions.json()["predictions"]
```

Comment:
You should handle error case here (e.g. call and catch `predictions.raise_for_status()`; see how it is done elsewhere in the code

Reply:
got it will update it.